bool setGet
uint8 flightMode

---

uint8 flightMode
